designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -40,17 +42,55 @@ object ManagementResources {
 
   import pl.touk.nussknacker.engine.api.CirceUtil._
 
+  // TODO: Why not just use the BestEffortJsonEncoder?
```

Comment:
I see that we use it but we also wrap the result with object with pretty the `pretty` field.

Reply:
Yeah. I realized that after a while.